miner/validatorenodes/LOG

```diff
@@ -0,0 +1,20 @@
+=============== Nov 15, 2019 (PST) ===============
```

Comment:
Are all these new files intentionally checked in?

Reply:
They were not. Thanks for catching that.